### ISSUE TITLE
Missing capitalisation in a reference: bug or misuse?

### ISSUE THREAD

**Issue by wilfwilson** [frankluebeck/GAPDoc#47]
In the Semigroups package's manual, there is an entry with a reference to the attribute `UnderlyingSemigroup` from the GAP reference manual. This the code used to make the reference https://github.com/semigroups/Semigroups/blob/5b25849d0c0d281fac83c216473e6f5d43b35acd/doc/isorms.xml#L79-L81:
```
<Ref Attr="UnderlyingSemigroup" Label="for a Rees 0-matrix semigroup" BookName="ref"/>
```

The compiled HTML version can be seen [here in the online version of the manual](https://semigroups.github.io/Semigroups/doc/chap18.html#X82B0BDCD7CBDCC2E), and I also attach a screenshot of the reference here:
<img width="789" alt="Screenshot 2021-05-20 at 17 44 43" src="https://user-images.githubusercontent.com/8956868/119017642-1e9f0080-b993-11eb-8ac7-8e87b43e855a.png">

The problem is that the 'R' of 'Rees 0-matrix semigroup' is not capitalised, even though:
* The label used in the reference to `UnderlyingSemigroup` uses a capital R (as shown above).
* The definition of the label in GAP's `doc/ref/reesmat.xml` uses a capital R: https://github.com/gap-system/gap/blob/567922f30ad821490ac0ff2d8f0cef28b2c93c02/doc/ref/reesmat.xml#L431-L434

Note that in the index of the GAP reference manual, there capitalisation is correct:
<img width="518" alt="Screenshot 2021-05-20 at 17 52 59" src="https://user-images.githubusercontent.com/8956868/119018707-480c5c00-b994-11eb-9bde-3b1d0c8acd2a.png">

The problem also exists in the PDF version of the manual.

Are we doing something wrong in the Semigroups package, or is this perhaps a bug in GAPDoc?

**Comment by frankluebeck** [comment]
Thanks for this example. The error comes from the GAP doc/ref/manual.six file; the wrong capitalization is also in the match list of the GAP help system.

I have fixed this in GAPDoc (07078d9ff065013d435651dbd5ec1071957ae81b).

To fix your example you have to rebuild the Reference manual in your GAP installation with an updated GAPDoc.

**Comment by wilfwilson** [comment]
This works nicely, thank you.
